dotCMS/src/integration-test/java/com/dotcms/security/apps/AppsAPIImplTest.java

```diff
@@ -91,122 +91,122 @@ public void Test_Store_Json_On_Different_Sites_Then_Pull_Sites_By_Host_Verify_Ma
         final Host host1 = new SiteDataGen().nextPersisted();
         final Host host2 = new SiteDataGen().nextPersisted();
 
-        final ServiceSecrets.Builder builder1 = new ServiceSecrets.Builder();
+        final AppSecrets.Builder builder1 = new AppSecrets.Builder();
 
-        final ServiceSecrets bean1Host1 = builder1.withServiceKey("serviceKey-1-Host-1")
+        final AppSecrets bean1Host1 = builder1.withKey("appKey-1-Host-1")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "appKey-1-Host-1" appears 10 times in this file; the first occurrence is on line 96](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5218098)